### ISSUE TITLE
Provide one docker image tag for both ARM and x86_64 architecture.

### ISSUE THREAD

**Issue by KEINOS** [KEINOS/Dockerfile_of_Alpine#1]
## TL;DR

> It would be nice if the image can be pulled by `keinos/alpine:latest` for both ARM and x86_64 architecture.

## TS;DR

Currently, this image (`keinos/alpine:latest`) provides only for x86_64 architecture. Because of the current specification of the automated build on DockerHub.

Since most of my images on Docker Hub are based on `keinos/alpine` image, I want them to be compatible with ARM and x86_64.

The `keinos/php8-jit` image [provides both](https://github.com/KEINOS/Dockerfile_of_PHP8-JIT/issues/2) (with different tags).

But to build ARM image, I had to rebuild the `keinos/alpine` first, then `keinos/php8-jit` on my RaspberryPi and then push it to Docker Hub.

If we can provide both ARM and x86_64 in one `keinos/alpine:latest` tag, it will ease me to rebuild the other images for ARM.

### How

- Prepare Manifest file for both architectures
  - REF:  [Docker Multi-Architecture Images](https://medium.com/@mauridb/docker-multi-architecture-images-365a44c26be6) @ Medium

- Receive webhook from Docker Cloud to RPi when the automated build succeeds. Then RPi builds and pushes the ARM image. 
  - The endpoint of RPi to receive the webhook, using `ngrok` or `serveo` might be nice and simple.
  - REF: [ngrokが便利すぎる](https://qiita.com/mininobu/items/b45dbc70faedf30f484e) @ Qiita
  - REF: [ngrokよりserveoがすごい。０秒で localhostを固定URLで公開](https://qiita.com/kaba/items/53b297e2bfb5b4f20a48) @ Qiita
